app/lib/multistep/controller.rb

```diff
@@ -0,0 +1,147 @@
+# frozen_string_literal: true
+
+module Multistep
+  module Controller
+    extend ActiveSupport::Concern
+
+    class_methods do
+      def form(value = nil, as: nil) # rubocop:disable Naming/MethodParameterName
```

Comment:
What is Rubocop unhappy about exactly? And can we avoid it somehow without disabling it?

Reply:
He's unhappy with `as` being too short. I could replace it with something like `alias:` but I think `as` is used often enough to be pretty standard.